specification/synapse/resource-manager/readme.md

```diff
@@ -73,6 +73,12 @@ directive:
     reason: Does not apply to sqlPool and bigDataPool as they are nested tracked resources
   - suppress: TrackedResourceListBySubscription
     reason: Does not apply to sqlPool and bigDataPool as they are nested tracked resources
+  - suppress: NestedResourcesMustHaveListOperation
```

Comment:
In this case, you'll need to follow the guidance like here: https://github.com/Azure/autorest/tree/97b68250afd96111f79047e24e22eeb82a30426f/src/autorest-core/test/variations/suppressions#example-limit-scope-to-the-one-document-namespace-yourapi-and-apply-the-filter-only-to-operationid-where-the-name-begins-with-legacy_

To add a suppression to only a specific path, first specify the from./where, then the action (suppress)